torch_xla/launch.py

```diff
@@ -0,0 +1,49 @@
+import warnings
+
+import torch.distributed as dist
+import torch_xla
+import torch_xla.core.xla_env_vars as xenv
+import torch_xla.distributed.xla_multiprocessing as xmp
+import torch_xla.utils.utils as xu
+import torch_xla.runtime as xr
+
+
+def spawn_wrapper(index, fn, *args, **kwargs):
+  dist.init_process_group('xla', init_method='xla://')
```

Comment:
PyTorch's documentation shows `init_process_group` inside of the user's main function, e.g. https://pytorch.org/tutorials/intermediate/dist_tuto.html

We should stick to that pattern and not `init_process_group` ourselves to be safe.